var/spack/repos/builtin/packages/harfbuzz/package.py

```diff
@@ -11,7 +11,7 @@ class Harfbuzz(MesonPackage, AutotoolsPackage):
     """The Harfbuzz package contains an OpenType text shaping engine."""
 
     homepage = "https://github.com/harfbuzz/harfbuzz"
-    url = "https://github.com/harfbuzz/harfbuzz/releases/download/2.9.1/harfbuzz-2.9.1.tar.xz"
+    url = "https://github.com/harfbuzz/harfbuzz/releases/download/9.0.0/harfbuzz-9.0.0.tar.xz"
```

Comment:
(Info) These changes not typically needed. Spack deduces the version-independent url from whatever url is there, regardless of whether it is the last version.